lib/Runtime/Library/WebAssemblyInstance.cpp

```diff
@@ -300,7 +300,14 @@ void WebAssemblyInstance::LoadImports(
                 AsmJsScriptFunction* func = AsmJsScriptFunction::FromVar(prop);
                 if (!wasmModule->GetWasmFunctionInfo(counter)->GetSignature()->IsEquivalent(func->GetSignature()))
                 {
-                    JavascriptError::ThrowWebAssemblyLinkError(ctx, WASMERR_SignatureMismatch);
+                    char16 temp[1024] = { 0 };
```

Comment:
If we already have a test case which catch this exception we should modify it print the message as it will cover this additional code as well. It is always a good idea to cover maximum code through unittests.